client/optimise.py

```diff
@@ -7,13 +7,16 @@
 
 class OptimiseClient(BaseClient):
     def __init__(
-        self, url: str, instance: str, client_id: str, client_secret: str
+            self, url: str, instance: str, client_id: str, client_secret: str
     ) -> None:
         super().__init__(url, instance, client_id, client_secret)
 
     def create_job(self, world_id: str, job: Dict[Any, Any]) -> requests.Response:
         return self._post(f"worlds/{world_id}/jobs", job).json()
 
+    def delete_job(self, world_id: str, job: str) -> requests.Response:
```

Comment:
Would be good to add a test to this (see https://github.com/ONSdigital/blaise-totalmobile-services/blob/main/tests/client/test_optimise.py )